src/lib/inc/bitmap.h

```diff
@@ -20,7 +20,7 @@ static const bitmap_granule_t ONE = 1;
 #define BITMAP_GRANULE_MASK(O, L)           BIT32_MASK((O), (L))
 
 #define BITMAP_SIZE(SIZE)                   (((SIZE) / BITMAP_GRANULE_LEN) + ((SIZE) % BITMAP_GRANULE_LEN ? 1 : 0))
-
+#define BITMAP_SIZE_IN_BYTES(NUM_BITS)      (((NUM_BITS) / 8) + (((NUM_BITS) % 8) > 0 ? 1 : 0))
```

Comment:
```suggestion
#define BITMAP_SIZE(NUM_BITS, BITS_PER_UNIT)                   (((NUM_BITS) / (??_WIDTH)) + ((NUM_BITS) % (BITS_PER_UNIT) ? 1 : 0))
#define BITMAP_SIZE_IN_GRANULE(NUM_BITS)    (BITMAP_SIZE((NUM_BITS), (BITMAP_GRANULE_LEN)))
#define BITMAP_SIZE_IN_BYTES(NUM_BITS)      (BITMAP_SIZE((NUM_BITS), 8))
```

And change where we used `BITMAP_SIZE` by `BITMAP_SIZE_IN_GRANULE`